frontend/src/store/modules/auth/actions.js

```diff
@@ -1,9 +1,24 @@
 import {LOGIN, LOGOUT} from './types/actions';
 import {SET_AUTHENTICATED_USER, USER_LOGIN, USER_LOGOUT} from "./types/mutations";
+import requestService from "../../../services/requestService";
 
 export default {
-    [LOGIN]: (context, user) => {
-        return new Promise((resolve, reject) => {
+    [LOGIN]: async (context, user) => {
+        const url = process.env.VUE_APP_API_URL + '/auth/login';
```

Comment:
Create config class with methods like getApiUrl, that will receive in constructor env params

Or define config file that exports functions like getApiUrl

In that way if necessary such file can be mocked for tests